plugin/src/main/scala/higherkindness/mu/rpc/srcgen/SrcGenPlugin.scala

```diff
@@ -155,7 +155,7 @@ object SrcGenPlugin extends AutoPlugin {
       }
     },
     muSrcGenCompressionType := NoCompressionGen,
-    muSrcGenIdiomaticEndpoints := false,
+    muSrcGenIdiomaticEndpoints := true,
```

Comment:
We'll probably need to update [this section](https://higherkindness.io/mu-scala/reference/source-generation#advanced-settings) in the mu-scala docs 👍

Reply:
Indeed, I've got a documentation PR for my-scala coming up.